package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@meowwolf/node-red-contrib-amqp",
-  "version": "0.2.0",
+  "version": "0.2.1",
   "description": "Amqp nodes for node-red",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec 'npm run build'",
-    "build": "rm -rf build/* && tsc -p . && npm run copyassets",
-    "build:production": "rm -rf build/* && tsc --sourceMap false -p . && npm run copyassets",
+    "start": "nodemon --exec npm run build",
+    "build": "rimraf build/* && tsc -p . && npm run copyassets",
+    "build:production": "rimraf build/* && tsc --sourceMap false -p . && npm run copyassets",
     "copyassets": "copyfiles -f src/nodes/**/*.html build/src/nodes && copyfiles -f src/nodes/icons/* build/src/nodes/icons",
     "lint": "eslint 'src/**'",
+    "lint:win": "eslint src/**",
```

Comment:
What does `lint:win` do here? Is it necessary? It just executes the same command as `lint`

Reply:
Oh. Never mind I see the missing single quotes now.

Reply:
I don't have a windows machine but I'm _pretty_ sure the change I just pushed up is the way to do it. From the eslint docs:
> Please note that when passing a glob as a parameter, it will be expanded by your shell. The results of the expansion can vary depending on your shell, and its configuration. If you want to use node glob syntax, you have to quote your parameter (using double quotes if you need it to run in Windows), as follows:
`eslint "lib/**"`

Let me know if that breaks things for you though.

Reply:
works well on windows. good catch, didn't see that one.